stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -768,11 +768,33 @@ function pinv(D::Diagonal{T}, tol::Real) where T
     Diagonal(Di)
 end
 
+# TODO Docstrings for eigvals, eigvecs, eigen all mention permute, scale, sortby as keyword args
```

Comment:
@dkarrasch Do you have an opinion on this?

Reply:
One would need to check if these keywords apply to all methods. I saw that you already implemented `sortby` in `eigen`. `scale` and `permute` don't apply to all methods, IIRC. I think they are specific to dense LAPACK matrices.

Reply:
I checked again and remembered that I added `scale, permute` to `eigen(::Diagonal{<:AbstractMatrix},...)`, because also `eigen(::Diagonal,...)` specifies them but doesn't actually use them. So I guess we could leave as it is, then they are just without effect.